### ISSUE TITLE
Heart Canisters are not baubles, making them not equipable

### ISSUE THREAD

**Issue by captinwill** [EmoKiba/Baubley-Heart-Canisters#47]
Using [baubley-heart-canisters-1.12.2-2.0.3.jar]

**Comment by Traverse-Joe** [comment]
Correct as of the 2.0 version of 1.12 I rewrote the mod and removed them from being baubles so you use the heart amulet to equip the canisters (recipe changed so it’s the first time you make)

**Comment by tails1974seeyounextgame** [comment]
yes but "heart necklace" is not a slot there is a "necklace" slot so the "heart necklace" is not equipabe o.0
